src/barplot.ts

```diff
@@ -34,6 +34,15 @@ export interface IBarPlotOptions extends IVisInstanceOptions {
    * @default NaN
    */
   max?: number;
+  /**
+   * scale such that the height matches the argument
+   * @default null
+   */
+  heightTo?: number;
+  /**
+   * @default 10
```

Comment:
Ups, copied comments (multiple times as you saw - will correct it and be more attentive next time :)